packages/react-aria-components/test/DropZone.test.js

```diff
@@ -404,5 +515,25 @@ describe('DropZone', () => {
 
       expect(await onDrop.mock.calls[0][0].items[0].getText('text/plain')).toBe('hello world');
     });
+
+    it('should not allow paste into the dropzone if disabled', async () => {
```

Comment:
I'm not sure the equivalent non-disabled version of this test would pass, we don't focus anything and that is an important part of the useClipboard logic

calling `await user.tab();` should be sufficient as the test above does

might be good to add a rerender to this test as well to remove the isDisabled and try pasting again (without trying to re-focus), this would verify the comment I had on Reid's thread

Reply:
I've fixed it in the following commit:
[Fix useClipboard disabled test](https://github.com/adobe/react-spectrum/pull/5958/commits/e24302ef60916d21889ba221ad92e12745b32d8e)